.github/runner/deployment.yaml.tmpl

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
```

Comment:
from the file name it's not exactly clear what this is used for. Should we add a comment / rename?

Reply:
Good point, I think we can do both here.